libnmstate/nm/team.py

```diff
@@ -100,6 +101,7 @@ def get_info(device):
 
 def _convert_teamd_config_to_nmstate_config(teamd_config, slave_names):
     teamd_config.pop(TEAMD_JSON_DEVICE, None)
+    teamd_config.pop(TEAMD_JSON_HWADDR, None)
```

Comment:
Will this remove 'hwaddr' in any cases? If so, I think now we want to remove 'hwaddr' from the output only when it is not explicitly set by user.  If people set it manually, it should be printed.

Reply:
This will remove the `hwaddr` always. This value is from teamd and is set automatically. In our case, it is useless because we are managing the MAC address in the interface state and not in team configuration. Please, note that we are doing the same with `device` as we do not want to show it because we are already managing the interface name without teamd.

IMHO, an user should not use those parameters and they should use the parameters provided by our API. Thanks.